propagation.go

```diff
@@ -169,6 +164,107 @@ func (p *textMapPropagator) Extract(abstractCarrier interface{}) (SpanContext, e
 	return ctx, nil
 }
 
+type zipkinB3Propagator struct {
```

Comment:
should I move the code to a new file `./zipkin/propagation.go` or add it to `zipkin.go`?